app/assets/stylesheets/exhibits/_header.scss

```diff
@@ -94,20 +94,23 @@ header {
 			}
 		}
 	}
+	.background-container {
+		height: 180px;
```

Comment:
These are the current recommendations on the masthead setup page:
"You can select and crop an image to use as a background in your exhibit site's masthead. To use an image as a masthead background, you should use an image that is at least 120 pixels tall and 1200 pixels wide. For best results use an image at least 1800 pixels wide. You can crop larger images using the cropping tool below."

Should these recommended dimensions be adjusted to match whatever you've found is actually the correct size?

Reply:
Updated the instructions based on what's in our Confluence documentation, which is more clear/accurate.